lib/Property/ICalendar/Recur.php

```diff
@@ -54,7 +54,7 @@ function setValue($value) {
                     if (strcmp($k, 'until') === 0) {
                         $v = strtr($v, [':' => '', '-' => '']);
                     }
-                } else {
+                } else if (!is_int($v)) {
```

Comment:
Hum if I am not wrong no it's is_int($v), is_string case is handle before, this else if(!is_int($v)) will fail if $v is a int (and not an array) for the int case, there nothing to do just store the value which is done outside the if else if.